napari/layers/base/base.py

```diff
@@ -411,13 +452,44 @@ def _mode_setter_helper(self, mode, Modeclass):
                 ),
             ),
         ]:
-            if mode_dict[old_mode] in callback_list:
-                callback_list.remove(mode_dict[old_mode])
+            if mode_dict[self._mode] in callback_list:
+                callback_list.remove(mode_dict[self._mode])
             callback_list.append(mode_dict[mode])
         self.cursor = self._cursor_modes[mode]
 
-        self.interactive = mode == Modeclass.PAN_ZOOM
-        return mode, True
+        self.interactive = mode == self._modeclass.PAN_ZOOM
+        self._overlays['transform_box'].visible = (
+            mode == self._modeclass.TRANSFORM
+        )
+
+        if mode == self._modeclass.TRANSFORM:
+            self.help = trans._(
+                'hold <space> to pan/zoom, hold <shift> to preserve aspect ratio and rotate in 45° increments'
+            )
+        elif mode == self._modeclass.PAN_ZOOM:
+            self.help = ''
+
+        return mode
+
+    @property
+    def mode(self) -> str:
+        """str: Interactive mode
+
+        Interactive mode. The normal, default mode is PAN_ZOOM, which
+        allows for normal interactivity with the canvas.
+
+        TRANSFORM allows for manipulation of the layer transform.
+        """
+        return str(self._mode)
+
+    @mode.setter
+    def mode(self, mode):
+        mode = self._mode_setter_helper(mode)
+        if mode == self._mode:
+            return
+        self._mode = mode
+
+        self.events.mode(mode=mode)
```

Comment:
```suggestion
        self.events.mode(mode=str(mode))
```
?
For type consistency.